nistats/regression.py

```diff
@@ -276,6 +276,7 @@ def __init__(self, theta, Y, model, wY, wresid, cov=None, dispersion=1.,
                                         dispersion, nuisance)
         self.wY = wY
         self.wresid = wresid
+        self.wdesign = model.wdesign
 
     @setattr_on_read
     def resid(self):
```

Comment:
@bthirion We should deprecate this and replace it with residuals in a separate PR, and others like it.
Or I should ask JD Kent to revert the changes I asked him to make and rename the resid mehotd in this PR back to `resid` so it stay consistent.
I like the former idea, renaming all uses of `resid` to `residuals`.
Does Nilearn have `resid` or `residuals`?

Reply:
Yes, let's do it in a sperate PR.
I'm not aware of `resid` in nilear? We should go for `residuals`

Reply:
Don't think Nilearn has any resid method.

Reply:
so we will keep the method as `resid` for this pull request, I will not change this